src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -34,6 +34,9 @@ object SampleWdl {
         |  output {
```

Comment:
No, I struggled a bit to identify the right place to add it. This was an earlier attempt.